src/test/ui/chalkify/bugs/async.rs

```diff
@@ -1,9 +1,14 @@
+// check-pass
+
+fn main() -> () {}
+/* FIXME: Add this back
```

Comment:
Oh, this can just be a check-fail, known-bug, with an ICE. I don't remember exactly which one, but there are at least some tests that expect and ICE, and we can use that.